README.md

```diff
@@ -33,6 +33,40 @@ python setup.py install --user
  speechmatics-flow --url $URL --auth-token $TOKEN --ssl-mode insecure
  ```
 
+### Change Default Assistant (Amelia → Humphrey)
+
+To set the default assistant to *Humphrey* instead of *Amelia* run this command:
+
+```bash
+speechmatics-flow --url $URL --auth-token $TOKEN --ssl-mode insecure --assistant humphrey
+```
+
+### Load conversation_config from a config file
+
+Instead of manually setting up conversation parameters, you can load them from a configuration file:
+
+1. Create the conversation_config.json File
+2. Dump the JSON template into a file named conversation_config.json and run flow client
+   using the `--config-file` option
```

Comment:
```suggestion
Instead of manually setting up conversation parameters, you can load them from a configuration file.

Create a JSON file with the template details, for example "conversation_config.json" and run flow client
using the `--config-file` option